packages/web3torrent/src/pages/file/File.tsx

```diff
@@ -9,8 +9,21 @@ import {parseURL, useQuery} from '../../utils/magnet';
 import {torrentStatusChecker} from '../../utils/torrent-status-checker';
 import {useInterval} from '../../utils/useInterval';
 import './File.scss';
-import WebTorrentPaidStreamingClient from '../../library/web3torrent-lib';
+import WebTorrentPaidStreamingClient, {TorrentTestResult} from '../../library/web3torrent-lib';
 import _ from 'lodash';
+import {Flash} from 'rimble-ui';
+
+const checkTorrent = async (web3Torrent, infoHash) => {
+  const testResult = await web3Torrent.checkTorrentInTracker(infoHash);
+  switch (testResult) {
+    case TorrentTestResult.NO_CONNECTION:
+      return `Your connection to the tracker may be limited, you might have unexpected functionality`;
+    case TorrentTestResult.NO_SEEDERS_FOUND:
+      return `Seems like the torrent doesn't have any seeders. You can give it a try nonetheless.`;
```

Comment:
```suggestion
      return 'Seems like the torrent doesn't have any seeders. You can give it a try nonetheless.';
```

Reply:
here this suggestion would make problems. because _doesn't_ has a `'`. I can change it to a `"` though

Reply:
Good catch! Updated the suggestion